app/src/main/java/com/kickstarter/services/DiscoveryParams.kt

```diff
@@ -560,20 +560,19 @@ class DiscoveryParams private constructor(
     enum class RaisedBuckets {
         BUCKET_0,
         BUCKET_1,
-        BUCKET_2,
-        UNKNOWN;
```

Comment:
Was generating a query error if the user pressed "See Results" on percentage raised screen